service-app/internal/auth/token.go

```diff
@@ -88,19 +50,17 @@ func (tg *jwtTokenGenerator) generateToken() (string, time.Time, error) {
 		return "", time.Time{}, fmt.Errorf("failed to sign token: %w", err)
 	}
 
-	expiry := time.Unix(claims.Exp, 0)
-
 	tg.logger.Info("Generated new JWT token.", nil)
```

Comment:
Not a blocker, but I don't think this log is particularly helpful and removing it would take out the whole dependency on the logger.